### PR TITLE
Add initial tests for Asset screen

### PR DESCRIPTION
**Pull request by grilix**
This sets up a basic test for the Asset screen

### CODE REVIEW

File: src/models/Transaction.js [hunk 1]
```diff
@@ -104,4 +104,5 @@ export type SmartWalletTransferTransaction = {
   hash: string,
   value: Object,
   asset: string,
+  status: string,
```
Comment (jsidorenko) [inline src/models/Transaction.js]: Why do you need this? For PPN transactions we don't have status, as they're getting confirmed immediately

File: src/utils/smartWallet.js [hunk 2]
```diff
@@ -23,10 +23,15 @@ import { TX_CONFIRMED_STATUS } from 'constants/historyConstants';
 
 import type { Accounts } from 'models/Account';
 import type { SmartWalletStatus } from 'models/SmartWalletStatus';
+import type { WalletReducerState } from 'reducers/smartWalletReducer';
```
Comment (jsidorenko) [inline src/utils/smartWallet.js]: I would rename the type to `SmartWalletReducerState`

File: src/models/Transaction.js (hunk 1 above)
Comment (grilix) [reply src/models/Transaction.js]: After typing, flow complains on this line: https://github.com/pillarwallet/pillarwallet/blob/42e49d3fcd3800e67dcea137125633fe784d2773/src/utils/smartWallet.js#L53

image

Any suggestion?

Comment (jsidorenko) [reply src/models/Transaction.js]: Cool, then leave the `status` field in typing

File: src/utils/smartWallet.js (hunk 2 above)
Comment (grilix) [reply src/utils/smartWallet.js]: Updated

**Pull request merged**